src/components/ModalWrapper.tsx

```diff
@@ -0,0 +1,54 @@
+import { Box, Button, Modal } from '@mui/material'
+import Divider from 'components/divider'
+import PageHeader from 'components/PageHeader'
+import { ModalProps } from 'types/custom_types'
+
+const style = {
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  bgcolor: 'background.paper',
+  border: '1px solid #000',
+  boxShadow: 24,
+  width: '50%',
+  p: 2,
+}
+
+export const ModalWrapper = (props: ModalProps) => {
+  return (
+    <>
+      <Modal
+        open={props.open}
+        onClose={props.onClose}
+        aria-labelledby="modal-modal-title"
+        aria-describedby="modal-modal-description"
+      >
+        <Box sx={style}>
+          <PageHeader title={props.header} description="" />
+          <Divider />
+          {props.content}
```

Comment:
There's a problem here and I'm not sure what the solution should be. On the command blocklist page's modal, if you were to choose to show 25 or more rows in the table, the result doesn't scroll and you can't get to the bottom of the content to change it back so you're stuck...